### ISSUE TITLE
control hyperparameters and number of epochs 

### ISSUE THREAD

**Issue by Liranbz** [hiredscorelabs/tamnun-ml#9]
Hi,
Thank you for this tool! How I can control on the number of the epochs or change the other hyperpararmeters?
Thank you!

**Comment by shudima** [comment]
Hi,
You can use `BertForClassification` module along with the `TorchEstimator` class like this:

```python
module = BertForClassification(output_size, bert_model_name, dropout=0.0, hidden_size=768)
cls = TorchEstimator(module).fit(X, y, batch_size=4, epochs=5)
```

**Comment by Liranbz** [comment]
Hi @shudima ,
for imbalanced data, is there a way to set weights in the loss function?
Thank you
